FileSpecs/Events.md

```diff
@@ -62,7 +62,7 @@ Party encounters a locked door.  The lock-picking difficulty seems to be indicat
 
 <!-- ---------------------------------------- -->
 ## Event 03: Message
-*Trigger*: Entering, if U03.3 is 0, or Looking if U03.3 is 1.  (Other values for U03.3: Unknown.)
+*Trigger*: Entering, if U03.3 is 0, or Looking if U03.3 is 1.  (Other values for U03.3: Unknown.  Following Amberstar semantics, `02` could be the flag for looking, and `00` is the default for "everything")
```

Comment:
Imo it is a bitfield. Bit 0 is for "enter tile" and bit 1 for "look".

So the values have the following meaning:

- 0: Enter tile or look (special case)
- 1: Enter tile
- 2: Look
- 3: Enter tile or look

Reply:
I double-checked the data and noticed a transcription error: we only see values `0`, `1`, or `2` in the existing maps.  I'll have to check some time if this is treated as a bit field.

Reply:
It is the same in Ambermoon. Only 0, 1 and 2. But I guess 0 is just the fallback for "allow both" as it won't make sense to disallow both. The event could never be triggered then.

Would be interesting if the value 3 would still allow both.

In the Ambermoon code I think it skips the check if the value is 0. Otherwise it does a bit test. So in Ambermoon 0 and 3 should both allow tile entering and looking as a trigger.

I would expect the same in Amberstar.

Reply:
`3` turns out to behave the same as `2`, so the logic seems to be a bit different.  I'll include this in the next revision.